web/index.html

```diff
@@ -488,13 +632,12 @@ <h3>Used in</h3>
                     }
                 });
 
-                // Preload initial chunks
-                for (let i = 0; i < 10; i++){
```

Comment:
I found that preloading chunks generally ended up slowing things down more than just loading them on the fly, since this chunk loading happens all at once while everything else is trying to load as well. Just loading chunks when you need them, and then handling it properly using async functions works very nicely and I have not noticed any slowdowns using this method.